omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderController1_10Test.java

```diff
@@ -589,4 +590,12 @@ public void invalidCareCenterShouldThrowException() throws Exception {
 		        );
 		handle(req);
 	}
+
```

Comment:
Is this change related to the pull request?

Reply:
No, it got mixed up from another branch, let me see how I can fix that